OpenRA.Mods.Common/SpriteLoaders/ShpTSLoader.cs

```diff
@@ -118,7 +118,9 @@ bool IsShpTS(Stream s)
 			while (w == 0 && h == 0 && f++ < imageCount);
 
 			s.Position = start;
-			return type < 4;
+
+			// Type 18 is a special case for RA2 low bridges.
```

Comment:
Sadly the same case here:
```
w 0 h 0 type 0
```

is the actual frame we want and it has _both_ width and height equal to zero.

Looping through all frames:
```
lobrdb01.sno
w 0 h 0 type 4
w 0 h 0 type 0
w 0 h 64256 type 18
w 0 h 0 type 0
w 30 h 19 type 120
```
none will pass the test even if we allow `w == 0 || h == 0` but don't accept `type == 18`.